### PR TITLE
TestExecBlock testcase implementation with valid/invalid counters

### PR DESCRIPTION
**Pull request by charltonh**
Updated PR

### CODE REVIEW

**Comment by adrianbrink** [comment]
@ebuchman Is this still realistic to be merged?

File: state/execution_test.go [hunk 1]
```diff
@@ -24,15 +29,125 @@ var (
...
+		}
+
+		n = i
+	}
+
+	// Potential errors
+	if n < blockCount {
+		t.Errorf("********************* saved %d blocks, expected %d **************", n, blockCount)
+	} else {
+		fmt.Println("********************* saved", n, "blocks out of", blockCount, "**************")
```
Comment (melekes) [inline state/execution_test.go]: `t.Logf`

File: state/execution_test.go [hunk 2]
```diff
@@ -24,15 +29,125 @@ var (
...
+	}
+
+	// Valid vs Invalid txns
+	fmt.Println("Issued", valtxns, "valid transactions, ", invtxns, "invalid transactions.")
```
Comment (melekes) [inline state/execution_test.go]: `t.Logf`

**Comment by ebuchman** [comment]
No, I think I wanted to leave open as reminder to rewrite these tests using tx results without events

**Comment by adrianbrink** [comment]
I'm closing this in favour of the issue #498.

**Pull request closed without merging**